SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security
+
+GitHub takes the security of our software products and services seriously, including all open source code repositories managed through our GitHub organizations, such as [GitHub](https://github.com/infraspecdev/).
```

Comment:
Change to:

Infraspec takes the security of our software products and services seriously, including all open-source code repositories managed through our GitHub organization, [Infraspec](https://github.com/infraspecdev/).

Reply:
Done d26b09f780dedbbfbe4dd7084e4186c29929ad4a